gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,149 @@
+              display the applicable license name.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h3>Nexus Page Header Style Classes</h3>
+      <p className="nx-p">
+        Note that <code className="nx-code">NxNexusPageHeader</code> shares much of its styling with
+        {' '}<code className="nx-code">NxPageHeader</code>. If you don't see the class you're looking for described
+        below check out <a href="#Pages/NxPageHeader/">NxPageHeader</a>.
+      </p>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row nx-table-row--header">
+            <th className="nx-cell nx-cell--header">Class</th>
+            <th className="nx-cell nx-cell--header">Location</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">.nx-page-header__extra-content-divider</td>
+            <td className="nx-cell">Child Element</td>
+            <td className="nx-cell">
+              Applied to a <code className="nx-code">&lt;div&gt;</code> to create a vertical rule between children
+              rendered in the extra content area at the right end of the header.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <p className="nx-p">
+        Note that the headers in the examples below have been tweaked to display in the normal page flow for the sake
+        of this.  Normally they would automatically display at the top of the viewport using CSS fixed positioning.
+      </p>
+      <p className="nx-p">
+        It is the responsibility of calling code to ensure the that content included in the header fits in a single
+        row at all supported resolutions. Behavior of this component when content exceeds the space allowed is
+        unspecified.
+      </p>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Nexus Page Header"
+                        id="nx-page-header-complex-example"
+                        codeExamples={nxNexusPageHeaderExampleCode}
+                        liveExample={NxNexusPageHeaderExample}
+                        defaultCheckeredBackground={true}>
+      An instance of <code className="nx-code">NxNexusPageHeader</code> with branding, navigation and examples of
+      extra content.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Nexus Page Header with custom logo"
+                        id="nx-page-header-complex-example"
+                        codeExamples={nxNexusPageHeaderCustomLogoExampleCode}
+                        liveExample={NxNexusPageHeaderCustomLogoExample}
+                        defaultCheckeredBackground={true}>
+      An instance of <code className="nx-code">NxNexusPageHeader</code> that demonstrates how to load a custom logo.
```

Comment:
Do we want to put this example first, as it is the expected usage, and then have the other example second demonstrating the "fallback" behavior?

Reply:
> Do we want to put this example first, as it is the expected usage, and then have the other example second demonstrating the "fallback" behavior?

Sure that makes sense: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/ba288f85a0d98a0dbc937040d86f28226f48b48f